draft-ietf-httpbis-client-hints.md

```diff
@@ -190,6 +200,12 @@ Implementers ought to consider both user and server controlled mechanisms and po
 Implementers SHOULD support Client Hints opt-in mechanisms and MUST clear persisted opt-in preferences when any one of site data, browsing history, browsing cache, or similar, are cleared.
 
 
+## Abuse Detection
+A user agent that tracks access to active fingerprinting information SHOULD consider emission of Client Hints headers similarly to the way it would consider access to the equivalent API.
+
```

Comment:
I was imagining both the "privacy budget" work as well as Mozilla's forensic work we discussed at IETF105 as potential ways to "track access to active fingerprinting information". I'm happy to drop the normative SHOULD here, if you think that'd be better. Alternatively, I can add examples pointing to those two methods of fingerprinting tracking.

Reply:
I think my point is broader than this: While the current client hints may be innocuous, this document does not provide any rubric for rationally evaluating abuse or for assessing new client hints, and this guidance doesn't help. So this just creates an open door for new dangerous features.

As I said in Montreal: I'm generally OK with taking features that are currently sent in headers and client-hintizing them, because that seems better. I'm not fine with a document that suggests that it is a good thing to take things that currently are acquired by active JS APIs and turn them into client hints.